### ISSUE TITLE
Reviews Incentives & Finalization

### ISSUE THREAD

**Issue by micahalcorn** [OriginProtocol/origin#1760]
In today's UX review, it became clear that we still are not completely settled on how to incentivize both buyers and sellers to leave reviews, protect the buyer from mistakenly releasing funds, and also minimize gas costs.

In our original purchase states, the review step was combined with each the buyer's confirmation of receipt and the seller's withdrawal of funds. As a result, both parties were required to leave a review containing at least a 1-5 star rating, and the number of blockchain transactions were reduced from four to two.

In the new marketplace contract, reviews are not required. The buyer's review is encouraged in the DApp by still having the review and finalization combined as one form submission, but the seller's review happens after the funds are already released. This presents two problems:

1. The buyer may instinctively leave a bad review and thereby unknowingly release the funds. We are addressing this for v1 by inferring the need for arbitration and intercepting the review process.
1. The seller has no incentive to return and leave a review. There is additional gas required AND the funds are already released.

Possible options are:

- require seller finalization after buyer finalization
- hold stake from both parties and release them in exchange for reviews
- settle for warning the buyer and create some incentive for the seller

This doesn't need to be settled for v1 but should be explored relatively soon.

cc @DanielVF @nick @matthewliu @joshfraser @wanderingstan 

Related: https://github.com/OriginProtocol/origin-js/issues/149 & https://github.com/OriginProtocol/origin-js/issues/365

**Comment by sparrowDom** [comment]
I agree that combining leaving the review and releasing the funds into one step (though it seemed like a practical solution at one point) is confusing. Approaches that make sense to me: 
- reduce the current finalisation window from 1 year to something less conservative (2-3 weeks perhaps?) and give the seller an option to withdraw the funds after that period. 
- would be nice if we were to have solid notifications so we can message the buyer and the seller that finalisation period is coming to an end in case the offer is not finalised
- I really like @nick 's idea of incentivising the buyer to leave a review by giving him some seller's OGN tokens. Because this has another benefit of also incentivising sellers to boost listings with OGN since that will not only expose their listings but also increase their chances to get more reviews from buyers.
